pkg/model/manifest_test.go

```diff
@@ -345,6 +354,30 @@ func Test_validateManifestBuild(t *testing.T) {
 			},
 			expectedErr: true,
 		},
+		{
+			name: "local dependency pointing to folder - error",
+			dependencies: ManifestDependencies{
+				"test": LocalDependency{
+					manifestPath: tmpTestSecretFile.Name(),
+				},
+			},
+		},
+		{
+			name: "local dependency pointing to non existen path - error",
+			dependencies: ManifestDependencies{
+				"test": LocalDependency{
+					manifestPath: filepath.Clean("/test/test/test"),
+				},
+			},
+		},
+		{
+			name: "local dependency pointing to correct file - no error",
```

Comment:
is there any difference between this and `local dependency pointing to folder - error` test case?

Reply:
yes, in the first one we are pointing to a folder that doesn't exists and on this one the folder exists